### ISSUE TITLE
Channel issue

### ISSUE THREAD

**Issue by Oscie57** [WiiLink24/service-issues#8]
1. What service did you run into this issue with?

Wii Room

2. What error did you encounter?

PPCHalt (on a PAL vWii)

3. How can we reproduce this issue?

I got a PPCHalt when pressing on the poster in the main room as it said "Thank You!" with the red table.

![image](https://user-images.githubusercontent.com/66192059/111530816-abdb9400-875b-11eb-9e57-5d447e21b060.png)

**Comment by SketchMaster2001** [comment]
I'm pretty sure Citrus already said this in the first issue

**Comment by spotlightishere** [comment]
This is true, thanks - added. I believe this to be an allocation-related issue as it can only be reproduced after clicking on the poster after watching a movie via the poster originally.

**Comment by scor57** [comment]
I believe this one was fixed. I'll need a tester to check.

**Comment by RengeLK** [comment]
Tested this with the latest beta WAD, still occurs.
